install.zasilkovna.php

```diff
@@ -130,12 +143,23 @@ public function postflight($route, JAdapterInstance $adapter) {
             }
 
 			$db = JFactory::getDBO();
-			$q = "CREATE TABLE IF NOT EXISTS #__virtuemart_zasilkovna_branches (
-										`id` int(10) NOT NULL,
-										`name_street` varchar(200) NOT NULL,
-										`currency` text NOT NULL,
-										`country` varchar(10) NOT NULL
-										) ENGINE=MyISAM DEFAULT CHARSET=utf8;";
+			$q = "CREATE TABLE `#__virtuemart_zasilkovna_carriers` (
```

Comment:
Zachoval bych IF NOT EXISTS.

Reply:
SQL se volá při instalaci. Já se potřebuji ujistit, že tabulka má takovou strukturu jakou jsem určil a né že si to někdo pozměnil a aplikace se tváří jako že to je v pořádku. Spiš bych tabulku dropnul pokud existuje. Ale to je defenzivní. Tabulka se nenuceně dropuje při odinstalaci.

Reply:
ok